### PR TITLE
Properly fetch the Y-m-d formatted date/times for event start/end

### PR DESCRIPTION
**Pull request by borkweb**
See: https://central.tri.be/issues/38864

### CODE REVIEW

File: src/Tribe/Template/Month.php [hunk 1]
```diff
@@ -394,26 +394,25 @@ protected function set_events_in_month() {
...
+				AND ( (tribe_event_start.meta_value >= '%1\$s' AND  tribe_event_start.meta_value <= '%2\$s')
+					OR (tribe_event_start.meta_value <= '%1\$s' AND tribe_event_end_date.meta_value >= '%1\$s')
+					OR ( tribe_event_start.meta_value >= '%1\$s' AND  tribe_event_start.meta_value <= '%2\$s')
+				)
+				AND $wpdb->posts.post_status IN('$post_stati')
+				ORDER BY $wpdb->posts.menu_order ASC, DATE(tribe_event_start.meta_value) ASC, TIME(tribe_event_start.meta_value) ASC;
+				",
+				$grid_start_datetime,
+				$grid_end_datetime
+			);
```
Comment (borkweb) [inline src/Tribe/Template/Month.php]: Note: this was just a whitespace change on this block.

1. removed extraneous spacing around the equal sign
1. Moved the `$wpdb->prepare` to the same line as the equal
1. Tabbed stuff back by one tab.

File: src/Tribe/Template/Month.php [hunk 2]
```diff
@@ -469,8 +468,8 @@ private function get_event_ids_by_day( $date ) {
 						}
 					}
 
-					$event_start = strtotime( tribe_get_start_date( $event->ID ) );
-					$event_end   = strtotime( tribe_get_end_date( $event->ID ) );
+					$event_start = strtotime( tribe_get_start_date( $event->ID, true, 'Y-m-d H:i:s' ) );
+					$event_end   = strtotime( tribe_get_end_date( $event->ID, true, 'Y-m-d H:i:s' ) );
```
Comment (barryhughes) [inline src/Tribe/Template/Month.php]: This is just an observation - personally, I think a string literal is far more readable here - but "should" we use the relevant date utils constant?

Comment (borkweb) [inline src/Tribe/Template/Month.php]: Hrm. I suppose that'd be appropriate.

**Comment by barryhughes** [comment]
Looks good :100:

**Pull request merged**